index.js

```diff
@@ -39,6 +56,31 @@ app.get("/messages", (req, res) => {
   res.json(messages.slice(afterId));
```

Comment:
We need a way to return the emojis associated with each message.

Reply:
Let me know if the  above makes sense to handle both the reaction duplications and return of emojis for each message

Reply:
fixed the DELETE endpoint and it works on my local server